### DISCUSSION TITLE
New icon auto move.

### DISCUSSION THREAD

**Discussion by MykeyRM** [Ross-Patterson/Portals-Desktop-Organization#346] [Ideas]

**Comment by Ross-Patterson** [comment]
There are tools that do this already, check out:

https://www.filejuggler.com/

https://github.com/tfeldmann/organize

http://www.dropitproject.com/

**Comment by Krisgiada** [comment]
It would be nice to have this integrated into the tool, most of the time it is needed for new installed programs that drop shortcuts on the desktop.  Automating the positioning of new icons using file type logic, without the need for other programs, would be useful.
